### PR TITLE
javascript-forms - Junou Lee

### PR DESCRIPTION
**Pull request by junoulee**
image
image

### CODE REVIEW

**Comment by junoulee** [comment]
image

**Comment by junoulee** [comment]
image

File: javascript-forms/form-values.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  var one = xx.elements.name.value;
+  var two = xx.elements.email.value;
+  var three = xx.elements.message.value;
+  var object = { name: one, email: two, message: three };
+
+  console.log(object);
+  xx.reset();
+}
+
+var xx = document.querySelector('#contact-form');
```
Comment (codingcodymiller) [inline javascript-forms/form-values.js:14]: You should make a better name for this variable.

File: javascript-forms/form-values.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+function getData(event) {
+  console.log('submit');
+
+  event.preventDefault();
+  var one = xx.elements.name.value;
+  var two = xx.elements.email.value;
+  var three = xx.elements.message.value;
+  var object = { name: one, email: two, message: three };
```
Comment (codingcodymiller) [inline javascript-forms/form-values.js:8]: these could use better names too. Get in the habit of giving your variables and functions quality names.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**